### DISCUSSION TITLE
My website is to show only chunks after few minutes

### DISCUSSION THREAD

**Discussion by ebennaga** [vercel/next.js#51217] [Help]
### Summary

![chunks](https://github.com/vercel/next.js/assets/2337413/e83ad9b5-3ffd-4c6b-b0c7-08a24ee30874)

I got this issue from my website and what should i do to fix it?

**Comment by icyJoseph** [comment]
Where is it hosted? 

- https://github.com/vercel/next.js/issues/48569
- https://github.com/vercel/next.js/pull/49140

**Reply by ebennaga** [reply]
aws amplify.
